clients/drcachesim/tests/offline-opcode_categories.templatex

```diff
@@ -25,8 +25,7 @@ Opcode mix tool results:
              16 :       lea
```

Comment:
Does this decoder change have the potential to cause backward-compatibility issues in drmemtrace? Do we need to increment OFFLINE_FILE_VERSION?

Reply:
> Fixes {v,}movntdqa which the above check detected had its operands in the wrong order: it was listed as a store in the table when it is in fact a load. 

I suppose instr_reads_memory and instr_writes_memory were both already correct? And the only change really is instr_get_category?

Maybe note that in the PR description.

Reply:
I don't think any version change is necessary: there is zero raw trace impact; the final trace is impacted for movntdqa and (for regdeps) the categories but there's no compatibility change: just an increase in accuracy.  raw2trace won't do that wrong thing; record_filter for regdeps won't do the wrong thing.

Reply:
Updated the descr.